backend/service/topology/cache.go

```diff
@@ -119,7 +119,7 @@ func (c *client) setCache(ctx context.Context, obj *topologyv1.Resource) error {
 	const upsertQuery = `
 		INSERT INTO topology_cache (id, resolver_type_url, data, metadata)
 		VALUES ($1, $2, $3, $4)
-		ON CONFLICT (id) DO UPDATE SET
+		ON CONFLICT (id, resolver_type_url) DO UPDATE SET
```

Comment:
when you deploy this the old queries will be running against the new schema. will they succeed?

Reply:
Operations on the topology cache will fail until the new code rolls out.
```
{"serviceName": "clutch.service.topology", "error": "pq: there is no unique or exclusion constraint matching the ON CONFLICT specification"}
```